SwrveSDK/src/main/java/com/swrve/sdk/locationcampaigns/LocationCampaignFilter.java

```diff
@@ -35,7 +34,8 @@
         try {
             wakeLock.acquire();
 
-            ((SwrveBase)(SwrveSDKBase.getInstance())).initLocationCampaigns(); // TODO: presumption for FA is that location campaigns have been refreshed from ABTS by starting the app. Fix for GA.
+            // TODO: presumption for FA is that location campaigns have been refreshed from ABTS by starting the app. Also that certain variables have been initialised. For FA hack this. For GA fix this.
```

Comment:
Is onCreate on the Application class called before this? if so we can move that initSDKForLocationCampaigns into the createInstance method